docs/usage/auto-updates.md

```diff
@@ -35,6 +35,26 @@ It is recommended to setup and monitor canary nodes, but otherwise normal worker
 
 The default and recommended configuration does not set any static wariness value on Zincati side, leaving rollout decisions to Cincinnati backend.
 
+## Strategies for updates finalization
+
+Zincati actively tries to detect and stage new updates whenever they become available.
+Once a new payload has been locally staged, a machine reboot is required in order to atomically apply the update to the system as a whole.
+
+Rebooting a machine does affect any workloads running on it at that time, and can potentially impact services across a whole cluster of nodes.
```

Comment:
nit: "Rebooting a machine affects workfoads running on the machine at that time, and ..."